src/networks.js

```diff
@@ -14,6 +15,7 @@ module.exports = {
     dustThreshold: 546 // https://github.com/bitcoin/bitcoin/blob/v0.9.2/src/core.h#L151-L162
   },
   testnet: {
```

Comment:
Isn't it implied that it's testnet3? Would naming it testnet3 be confusing? Don't a significant majority use testnet3 as opposed to any previous? I don't have a strong preference on the naming.